clicommand/build_cancel.go

```diff
@@ -0,0 +1,114 @@
+package clicommand
+
+import (
+	"context"
+	"fmt"
+
+	"time"
+
+	"github.com/buildkite/agent/v3/api"
+	"github.com/buildkite/agent/v3/logger"
+	"github.com/buildkite/roko"
+	"github.com/urfave/cli"
+)
+
+const buildCancelDescription = `Usage:
+
+    buildkite-agent build cancel [options...]
+
+Description:
+
+Cancel a running build.
+
+Example:
+
+    $ buildkite-agent build cancel --build "1234"`
```

Comment:
Maybe make it clear this should work within the current build (if there is one)?

```suggestion
    # Cancels the current build
    $ buildkite-agent build cancel
    # Cancel a different build
    $ buildkite-agent build cancel --build "01234567-89ab-cdef-0123-456789abcdef"`
```